builtins/amp-pixel.js

```diff
@@ -69,9 +70,29 @@ export class AmpPixel extends BaseElement {
           .expandAsync(this.assertSource_(src))
           .then(src => {
             const image = new Image();
-            image.src = src;
-            dev().info(TAG, 'pixel triggered: ', src);
-            return image;
+            const referrerPolicy = this.element.getAttribute('referrerpolicy');
+            // if "referrerPolicy" is not supported, use iframe wrapper
+            // to scrub the referrer.
+            if (image.referrerPolicy === undefined &&
+                referrerPolicy == 'no-referrer') {
+              const iframe = createElementWithAttributes(
+                  this.win.document, 'iframe', {
+                    src: `javascript: '<img src="${src}">'`,
+                  });
+              this.element.appendChild(iframe);
+              dev().info(TAG, 'pixel triggered via iframe: ', src);
+              return iframe;
+            } else {
+              if (referrerPolicy) {
+                image.referrerPolicy = referrerPolicy;
+                // referrerPolicy is respected only if the image is attached
```

Comment:
behavior confirmed. 
but don't see it documented in the [spec](https://www.w3.org/TR/referrer-policy/#referrer-policy-delivery-referrer-attribute). could be a bug. not sure what to expect